.github/workflows/action-test.yml

```diff
@@ -0,0 +1,44 @@
+name: Action test
+
+on:
+  push:
+    branches: [main, release/**]
```

Comment:
We should be running this on pull request as well.

Reply:
Hmm yeah actually I think it should also run after build jobs and using correct ref from the hub

Reply:
It could do that, but it's actually good in some ways to use the current latest, because that's the way we should make changes to the action. Changes to the action should probably never occur in lockstep with the image, otherwise people will see breakages.